tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+      "lib": ["esnext"],
```

Comment:
For my personal knowledge, what is this lib for?

Reply:
management of the flatMap in ts that is only available since ES2019

Reply:
Thanks :)